test/components/modal.spec.js

```diff
@@ -305,4 +342,156 @@ describe('Modal spec', () => {
       expect(!instance.$modal.hasClass('modal-content-leave')).to.be.true
     }))
   })
+
+  describe('onModalClick', () => {
+    let fakeEvent
+
+    context('when event target is equal $modal', () => {
+      beforeEach(() => {
+        let modal = $('<div></div>')
+        instance.$modal = modal
+
+        fakeEvent = {
+          target: modal
+        }
+      })
+
+      context('when static option is set to false', () => {
```

Comment:
maybe, theses contexts shouldn't be inside `context('when event target is equal $modal')`, is it right?